html/dom/documents/dom-tree-accessors/nameditem-fetch-01.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>Named items: fetch text/html</title>
+<link rel="author" title="bzbarsky" href="mailto:bzbarsky@mit.edu">
```

Comment:
How did bz author these?

Reply:
I used @bzbarsky’s code from https://github.com/whatwg/html/issues/4792#issue-471167003 as&nbsp;a&nbsp;base.

I&nbsp;only&nbsp;changed the&nbsp;`alert`&nbsp;call into&nbsp;`assert_*`&nbsp;calls and&nbsp;added the&nbsp;**WPT**&nbsp;boilerplate.